include/openenclave/internal/defs.h

```diff
@@ -7,8 +7,21 @@
 #include <openenclave/bits/defs.h>
 
 /* OE_WEAK_ALIAS */
+#ifdef __GNUC__
 #define OE_WEAK_ALIAS(OLD, NEW) \
     extern __typeof(OLD) NEW __attribute__((__weak__, alias(#OLD)))
+#elif _MSC_VER
+#define OE_WEAK_ALIAS __pragma(comment(linker, "/alternatename:" #NEW "=" #OLD))
```

Comment:
This should be `OE_WEAK_ALIAS(OLD, NEW)`. This is updated in my local copy but I forgot to push the change